react-components/package.json

```diff
@@ -1,19 +1,25 @@
 {
-  "name": "minicart",
+  "name": "@adobe/aem-core-cif-react-components",
```

Comment:
Can we keep the version of the `@adobe/aem-core-cif-react-components` npm project in sync with the maven reactor project version?

Reply:
Good point, I think this could be done in the POM by adding a an extra step `npm-module-prepare` that would do `npm version ${project.version}`, and would be bound to the right phase/goal during `release:prepare`. I think you could use the `completionGoals` option of the `maven-release-plugin` to do that at the right time.

Reply:
@cjelger @herzog31, how "blocking" is this issue for us now? I've been trying to find a suitable solution for a few days now and nothing seems to work properly. Maven's `completionGoals` runs only Maven goals and doesn't run `scm:add` and `scm:commit` properly. I even created my own `release.js` script to handle the version update and Git commit, but that is not executed correctly by the `exec-maven-plugin` (can't even see the output from it).

For this release, we can sync the version manually (i.e. just put `0.6.0` in the version field) and continue the work after this Friday's release, what do you think?

Reply:
Sure no problem, we can fix it later, it's not important for this release.

Reply:
👍

Reply:
I added a script to automatically update the version of the package and commit the changes. This script can be ran manually via `npm run update-version -- --version=x.x.x`. I tried to convince Maven to run it but I couldn't get the right version, it would always call it with the `SNAPSHOT`